data/tools/GUI.pyw

```diff
@@ -42,6 +45,13 @@ APP_DIR,APP_NAME=os.path.split(os.path.realpath(sys.argv[0]))
 WESNOTH_ROOT_DIR=os.sep.join(APP_DIR.split(os.sep)[:-2])  # pop out "data" and "tools"
 WESNOTH_DATA_DIR=os.path.join(WESNOTH_ROOT_DIR,"data")
 WESNOTH_CORE_DIR=os.path.normpath(os.path.join(WESNOTH_DATA_DIR,"core"))
+WESNOTH_TRAN_DIR=os.path.join(WESNOTH_ROOT_DIR, "translations")
+
+try:
+    SYSTEM_LOCALE = locale.getdefaultlocale()[0]
```

Comment:
Triggers a deprecation warning with Python 3.11, this is going to be removed in Python 3.13 (python/cpython#90817).

The warning says "Use setlocale(), getencoding() and getlocale() instead"

Reply:
Unfortunately, ``getlocale()`` is unusable in Windows, and [it's also on the road of deprecation](https://bugs.python.org/issue43557). Until an actual viable alternative exists, this is probably the best option to get well, the _default_ locale.

Reply:
Thanks for improving the code.

Minor correction: `getlocale()` doesn't seem to be on the road to deprecation. That's proposed in the first message of that thread, however it gets voted against, and the same person who proposed it then adds the deprecation message for `getdefaultlocale()`.